src/compiler/transformers/ts.ts

```diff
@@ -1938,14 +1938,14 @@ namespace ts {
 
             resumeLexicalEnvironment();
 
-            const indexAfterLastPrologueStatement = factory.copyPrologue(body.statements, statements, /*ensureUseStrict*/ false, visitor);
-            const superStatementIndex = findSuperStatementIndex(body.statements, indexAfterLastPrologueStatement);
+            const prologueStatementCount = factory.copyPrologue(body.statements, statements, /*ensureUseStrict*/ false, visitor);
+            const superStatementIndex = findSuperStatementIndex(body.statements, prologueStatementCount);

```

Comment:
I'm nitpicking the variable name here. Both variable names are correct; `prologueStatementCount` just makes more sense to me -- especially in the context of the new array spread below.